scripts/start_e2eaiok_docker.py

```diff
@@ -295,25 +295,29 @@ def main(input_args):
 
     hostname = os.uname()[1]
     print_success = False
-    if input_args.backend == 'pytorch':
-        docker_name = "e2eaiok-pytorch110"
-        docker_file = "DockerfilePytorch110"
-        docker_nickname = "e2eaiok-pytorch"
-        port = 12345
+    # latest tensorflow env
     if input_args.backend == 'tensorflow':
-        docker_name = "e2eaiok-tensorflow210"
-        docker_file = "DockerfileTensorflow210"
+        docker_name = "e2eaiok-tensorflow"
+        docker_file = "DockerfileTensorflow"
         docker_nickname = "e2eaiok-tensorflow"
         port = 12344
-    if input_args.backend == 'pytorch_mlperf':
+    # pytorch1.10 env
+    if input_args.backend == 'pytorch110':
```

Comment:
if pytorch120 is working for all models instead of dlrm, we don't need to keep pytorch110 here.

Reply:
pytorch120 is for DeNas.
pytorch1.10 base image has some bugs, rnnt cannot use it, and pdt performance is based on pytorch1.10

Reply:
No. It cannot work for all models. PyTorch1.12 base image intel/oneapi-aikit:2022.3.0-devel-ubuntu18.04 has bugs with torch_ccl.

Reply:
We need to keep pytorch1.10 for RNNT, because pytorch1.10 base image works and pytorch1.12 base image has bugs.

Reply:
PyTorch1.10 is the latest workable version in docker env.
![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/8370601/202964443-0ceffea6-1b63-4b26-9f21-9d28a935f996.png)